standard/attributes.md

```diff
@@ -341,14 +360,18 @@ It is a compile-time error to use a single-use attribute class more than once on
 > *Example*: The example
 > ```csharp
 > using System;
+>
 > [AttributeUsage(AttributeTargets.Class)]
-> public class HelpStringAttribute: Attribute
+> public class HelpStringAttribute : Attribute
 > {
 >     string value;
->     public HelpStringAttribute(string value) {
+>     public HelpStringAttribute(string value)
+>     {
 >         this.value = value;
 >     }
->     public string Value { 
+>
+>     public string Value
+>     { 
 >             get {...} 
```

Comment:
Even if we keep this on a line on its own, the indentation looks wrong - maybe it's using a tab instead of spaces?